src/Microsoft.Metadata.Visualizer.Tests/MdvArgumentsTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I don't think we need a test for the command line args at this point. Especially since this is a trivial on/off switch.

We should migrate to System.CommandLine APIs anyways.